dali/ft/filecopy.cpp

```diff
@@ -2873,6 +2910,16 @@ void FileSprayer::updateTargetProperties()
                 // TODO: Create DistributedFilePropertyLock for parts
                 curPart->lockProperties();
                 IPropertyTree& curProps = curPart->queryAttributes();
+
+                FilePartInfo & curHeaderSource = sources.item(whichHeaderInput);
+                curProps.setPropInt(FAheaderLength, curHeaderSource.xmlHeaderLength);
+
+                FilePartInfo & curFooterSource = sources.item(cur.whichInput);
+                curProps.setPropInt(FAfooterLength, curFooterSource.xmlFooterLength);
+
+                if( idx+1 != partition.ordinality() && partition.item(idx+1).whichOutput != cur.whichOutput )
```

Comment:
odd spacing.
if "idx+1 != partition.ordinality()" is true, you don't need to test "partition.item(idx+1).whichOutput != cur.whichOutput" here, as must be true from line 2907 already.